test/integration/esys-create-password-auth.int.c

```diff
@@ -0,0 +1,329 @@
+                 .size = 0,
+                 .buffer = {0}
+             },
+            .data = {
+                 .size = 0,
+                 .buffer = {}
+             }
+        }
+    };
+
+    inSensitive2.sensitive.userAuth = authKey2;
+
+    TPM2B_SENSITIVE_CREATE inSensitive3 = {
+        .size = 1,
+        .sensitive = {
+            .userAuth = {
+                 .size = 0,
+                 .buffer = {}
+             },
+            .data = {
+                 .size = 0,
+                 .buffer = {}
+             }
+        }
+    };
+
+    TPM2B_PUBLIC inPublic2 = {
+        .size = 0,
+        .publicArea = {
+            .type = TPM2_ALG_RSA,
+            .nameAlg = TPM2_ALG_SHA256,
+            .objectAttributes = (TPMA_OBJECT_USERWITHAUTH |
+                                 TPMA_OBJECT_RESTRICTED |
+                                 TPMA_OBJECT_DECRYPT |
+                                 TPMA_OBJECT_FIXEDTPM |
+                                 TPMA_OBJECT_FIXEDPARENT |
+                                 TPMA_OBJECT_SENSITIVEDATAORIGIN),
+
+            .authPolicy = {
+                 .size = 0,
+             },
+            .parameters.rsaDetail = {
+                 .symmetric = {
+                     .algorithm = TPM2_ALG_AES,
+                     .keyBits.aes = 128,
+                     .mode.aes = TPM2_ALG_CFB
+                 },
+                 .scheme = {
+                      .scheme =
+                      TPM2_ALG_NULL,
+                  },
+                 .keyBits = 2048,
+                 .exponent = 65537
+             },
+            .unique.rsa = {
+                 .size = 0,
+                 .buffer = {}
+                 ,
+             }
+        }
+    };
+
+    TPM2B_DATA outsideInfo2 = {
+        .size = 0,
+        .buffer = {}
+        ,
+    };
+
+    TPML_PCR_SELECTION creationPCR2 = {
+        .count = 0,
+    };
+
+    TPM2B_PUBLIC *outPublic2;
+    TPM2B_PRIVATE *outPrivate2;
+    TPM2B_CREATION_DATA *creationData2;
+    TPM2B_DIGEST *creationHash2;
+    TPMT_TK_CREATION *creationTicket2;
+
+    r = Esys_Create(esys_context,
+                    primaryHandle_handle,
+                    ESYS_TR_PASSWORD, ESYS_TR_NONE, ESYS_TR_NONE,
+                    &inSensitive2,
+                    &inPublic2,
+                    &outsideInfo2,
+                    &creationPCR2,
+                    &outPrivate2,
+                    &outPublic2,
+                    &creationData2, &creationHash2, &creationTicket2);
+    goto_if_error(r, "Error esys create ", error);
+
+    fprintf(stderr, "\nSecond key created.\n");
+
+    ESYS_TR loadedKeyHandle;
+
+    r = Esys_Load(esys_context,
+                  primaryHandle_handle,
+                  ESYS_TR_PASSWORD,
+                  ESYS_TR_NONE,
+                  ESYS_TR_NONE, outPrivate2, outPublic2, &loadedKeyHandle);
+    goto_if_error(r, "Error esys load ", error);
+
+    fprintf(stderr, "\nSecond Key loaded.\n");
```

Comment:
Use the functions provided by the logging framework instead of calling fprintf() directly.
Here and below.